.vscode/settings.json

```diff
@@ -128,8 +128,8 @@
   // C++/Circuits settings
```

Comment:
These changes probably sneaked in here by accident. Once you remove these changes. I'll approve it.